<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gacha_5_times</class>
 <widget class="QWidget" name="gacha_5_times">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>抽卡！！！！！</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pictures.qrc">
    <normaloff>:/pictures/logo.png</normaloff>:/pictures/logo.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>591</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>111</width>
           <height>411</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_1">
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MyPushButton" name="pushButton_1">
            <property name="styleSheet">
             <string notr="true">color:rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>确认</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <widget class="QWidget" name="layoutWidget_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>111</width>
           <height>411</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MyPushButton" name="pushButton_2">
            <property name="styleSheet">
             <string notr="true">color:rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>确认</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>111</width>
           <height>411</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MyPushButton" name="pushButton_3">
            <property name="styleSheet">
             <string notr="true">color:rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>确认</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <widget class="QWidget" name="layoutWidget_3">
         <property name="geometry">
          <rect>
           <x>1</x>
           <y>10</y>
           <width>111</width>
           <height>411</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MyPushButton" name="pushButton_4">
            <property name="styleSheet">
             <string notr="true">color:rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>确认</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_6" native="true">
        <widget class="QWidget" name="layoutWidget_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>111</width>
           <height>411</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MyPushButton" name="pushButton_5">
            <property name="styleSheet">
             <string notr="true">color:rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>确认</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="Kong">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>剩余行动力:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="move">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>(action)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MyPushButton" name="goto_widget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(255, 255, 255)</string>
         </property>
         <property name="text">
          <string>不选了</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="kong">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">mypushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="pictures.qrc"/>
 </resources>
 <connections/>
</ui>
